exaudfclient/base/.bazelrc

```diff
@@ -17,7 +17,7 @@ build:test-binaries-py2 --config=static-binary-py2 --config=slow-wrapper-py2
 build:test-binaries-both --config=test-binaries-py2 --config=test-binaries-py3
 build:verbose --copt='-v' --subcommands --verbose_failures --announce_rc
 #TODO test linkopts="-flto"
-build:optimize --copt="-g0" --copt="-DNDEBUG" --copt=-fstack-protector-strong --copt=-fomit-frame-pointer --copt=-ffunction-sections --copt=-fdata-sections --copt="-O3" --copt="-U_FORTIFY_SOURCE" --copt="-flto"
+build:optimize --copt="-g0" --copt="-DNDEBUG" --copt=-fstack-protector-strong --copt=-fomit-frame-pointer --copt=-ffunction-sections --copt=-fdata-sections --copt="-O3" --copt="-U_FORTIFY_SOURCE" --copt="-flto" --copt="-fuse-linker-plugin"
```

Comment:
Any reason to add this? From what I have seen on stackoverflow, this option is automatically enabled on newer versions of gcc 
(https://stackoverflow.com/questions/15606993/g-optimization-flags-fuse-linker-plugin-vs-fwhole-program)

Reply:
good point, this is actually something we need to check. We get weird messages regarding lto during build. I need to investigate this more and also run performance tests to verify we didn't get slower.